Crashlytics/Crashlytics/Components/FIRCLSApplication.m

```diff
@@ -48,6 +48,8 @@
   return @"tvos";
 #elif TARGET_OS_WATCH
   return @"ios";  // TODO: temporarily use iOS until Firebase can add watchos to the backend
```

Comment:
@themiswang Any update on the backend changes for watchos and visionos?

Reply:
Thanks for checking, I'll leave the TODOs for now.